#1. Membuat DataFrame

In [1]:
import pandas as pd
import numpy as np

# Data penjualan dari gambar yang telah diidentifikasi
data = [100, 150, 50, 100, 130, 120, 100, 85, 70, 150,
                   150, 120, 50, 100, 100, 140, 90, 150, 50, 90,
                   120, 100, 110, 75, 65]

# Membuat DataFrame 1 kolom dengan 25 baris
df = pd.DataFrame(data, columns=['TransactionAmount'])

df


TransactionAmount
0                 100
1                 150
2                  50
3                 100
4                 130
5                 120
6                 100
7                  85
8                  70
9                 150
10                150
11                120
12                 50
13                100
14                100
15                140
16                 90
17                150
18                 50
19                 90
20                120
21                100
22                110
23                 75
24                 65

#2. Analisis Measure of Central Tendency

In [3]:
# Analisis measure of central tendency
mean_value = df['TransactionAmount'].mean()
median_value = df['TransactionAmount'].median()
mode_value = df['TransactionAmount'].mode()[0]

mean_value, median_value, mode_value



(102.6, 100.0, 100)

#3. Analisis Measure of Variability

In [4]:
# Analisis measure of variability
variance_value = df['TransactionAmount'].var()
std_dev_value = df['TransactionAmount'].std()
range_value = df['TransactionAmount'].max() - ['TransactionAmount'].min()
quartiles_value = df['TransactionAmount'].quantile([0.25, 0.5, 0.75])


variance_value, std_dev_value, range_value, quartiles_value

(1012.7499999999999,
 31.823733281939123,
 100,
 0.25     85.0
 0.50    100.0
 0.75    120.0
 Name: TransactionAmount, dtype: float64)

#5. Menentukan H0 dan H1 T-Test

H0 : Penjualan rata-rata perusahaan X = $100

H1 : Penjualan rata-rata perusahaan X > $100


#Melakukan T-Test:

In [ ]:
from scipy import stats

# Melakukan T-Test
t_stat, p_value = stats.ttest_1samp(df['TransactionAmount'], 100)

t_stat, p_value

(0.4085001556802841, 0.6865284813438117)

Dengan hasil T-Test sebagai berikut:

*  Nilai t-statistik (t_stat): 0.408
*  Nilai p-value (p_value): 0.687

**Interpretasi Hasil:**

P-value:

*   Nilai p-value sebesar 0.687 jauh lebih besar dari tingkat signifikansi yang umum digunakan (α = 0.05).
*   Karena p-value > 0.05, kita gagal menolak H0.

Kesimpulan:

Tidak ada bukti yang cukup untuk mengatakan bahwa rata-rata jumlah transaksi setelah training berbeda secara signifikan dari $100.
Ini berarti training yang dilakukan kepada pekerja sales mungkin tidak memiliki dampak signifikan terhadap peningkatan rata-rata penjualan.